docs/install.md

```diff
@@ -234,3 +234,18 @@ At the current time, leftover data in `/` is **NOT** automatically cleaned up.
 be useful, because it allows the new image to automatically import data from the previous
 host system!  For example, things like SSH keys or container images can be copied
 and then deleted from the original.
+
+### Using `bootc install to-filesystem --source-imgref <imgref>`
+
+By default, `bootc install` has to be run inside a podman container. With this assumption,
+it can escape the container, find the source container image (including its layers) in
+the podman's container storage and use it to create the image.
+
+When `--source-imgref <imgref>` is given, `bootc` no longer assumes that it runs inside podman.
+Instead, the given container image reference (see [skopeo(1)](https://github.com/containers/skopeo/blob/main/docs/skopeo.1.md)
```

Comment:
Can we point to https://github.com/containers/image/blob/main/docs/containers-transports.5.md instead?  Are all of the transports supported?

Reply:
Thanks!  After quick glimpse at the skopeo-proxy code it seems it supports all transports.  @cgwalters, do you confirm?

Reply:
Yep, it does